mscore/plugin/api/tie.h

```diff
@@ -0,0 +1,58 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2019 Werner Schweer and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __PLUGIN_API_TIE_H__
+#define __PLUGIN_API_TIE_H__
+
+#include "elements.h"
+#include "libmscore/tie.h"
+#include "libmscore/note.h"
+
+namespace Ms {
+namespace PluginAPI {
+
+//---------------------------------------------------------
+//   Tie
+///  Provides access to internal Ms::Tie objects.
+///  \since MuseScore 3.3
+//---------------------------------------------------------
+
+class Tie : public QObject {
```

Comment:
It would be better to inherit this `Tie` from `Element`, then it would inherit other relevant properties too (like `parent`, `tick`, `track`, `autoplace` etc.)